draft-ietf-tls-tls13.md

```diff
@@ -273,10 +273,6 @@ that has two basic properties:
 - The connection is reliable.  Messages include an authentication
   tag which protects them against modification.
 
-Note: The TLS Record Protocol can operate in an insecure mode but is generally
-only used in this mode while another protocol is using the TLS Record
-Protocol as a transport for negotiating security parameters.
-
```

Comment:
Maybe just replace with a short sentence in here somewhere stating that the handshake starts in the clear, it switches to encrypted transport as soon as possible, and is retroactively authenticated before handshake completion?